ci-operator/step-registry/rosa/terraform/aws/e2e/test/rosa-terraform-aws-e2e-test-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+CLOUD_PROVIDER_REGION=${LEASED_RESOURCE}
+
+# Configure aws
+AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
+if [[ -f "${AWSCRED}" ]]; then
+  export AWS_SHARED_CREDENTIALS_FILE="${AWSCRED}"
+  export AWS_DEFAULT_REGION="${CLOUD_PROVIDER_REGION}"
+else
+  echo "Did not find compatible cloud provider cluster_profile"
+  exit 1
+fi
+
+
+mkdir -p ~/.aws
+cp ${AWSCRED} ~/.aws/credentials
```

Comment:
If `terraform` supports getting the credentials from the above two AWS variables, we may not need to do this.

Reply:
"terraform supports getting the credentials from the above two AWS variables" - what does it mean?
Running this terraform code requires AWS credentilas (taking it from ~/.aws/credentials)

Reply:
For rosa cli, if the variables `AWS_SHARED_CREDENTIALS_FILE` and `AWS_DEFAULT_REGION` are set, it will read credentials from the variables, and do not need to config `~/.aws`.